webapp/components/user_settings/user_settings_general.jsx

```diff
@@ -936,6 +963,99 @@ class UserSettingsGeneralTab extends React.Component {
             );
         }
 
+        let positionSection;
+        if (this.props.activeSection === 'position') {
+            let extraInfo;
+            let submit = null;
+            if ((this.props.user.auth_service === 'ldap' || this.props.user.auth_service === Constants.SAML_SERVICE) && global.window.mm_config.PositionAttributeSet === 'true') {
```

Comment:
@yangchen1 sorry, there isn't actually a question there! It was "Should the position field be populated automatically from external auth providers, like other fields are?". I think we've come to the conclusion that the answer is yes.